src/network/netdev/tunnel.c

```diff
@@ -617,6 +692,38 @@ int config_parse_encap_limit(const char* unit,
         return 0;
 }
 
+int config_parse_erpsan_index(const char* unit,
+                             const char *filename,
+                             unsigned line,
+                             const char *section,
+                             unsigned section_line,
+                             const char *lvalue,
+                             int ltype,
+                             const char *rvalue,
+                             void *data,
+                             void *userdata) {
+        Tunnel *t = userdata;
+        uint32_t k = 0;
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+
+        r = safe_atou32(rvalue, &k);
+        if (r < 0) {
+                log_syntax(unit, LOG_ERR, filename, line, r, "Failed to parse erspan index option, ignoring: %s", rvalue);
+                return 0;
+        }
+
+        if (k & ~((1<<20) - 1) || k == 0)
```

Comment:
It seems OK. But I prefer `k >= (1 << 20)`.

Reply:
ok